src/main/java/baseball/model/GameResult.java

```diff
@@ -0,0 +1,39 @@
+package baseball.model;
+
+public class GameResult {
+
+    public GameResultData result(int strike, int ball) {
```

Comment:
`StrikeBallResultDto` 객체로 `strike` 와 `ball` 을 묶어 만드셨는데,
스트라이크와 볼을 매개변수로 받을 때 활용했으면 좋았을 것 같아요!

```suggestion
    public GameResultData result(StrikeBallResultDto strikeBallResultDto) {
```